RFC-005-SWAP-Basic-Bitcoin.adoc

```diff
@@ -0,0 +1,310 @@
+= Bitcoin Basic HTLC Atomic Swap
+:toc:
+:revdate: 2019-03-01
+
+NOTE: RFC-Number: 005 +
+Status: Draft +
+Discussion-issue: https://github.com/comit-network/RFCs/issues/42[#42] +
++ Created on: {revdate} +
+
+== Description
+
+This RFC defines how to execute a link:./RFC-003-SWAP-Basic.md[RFC003] SWAP where one of the ledgers is Bitcoin and the associated asset is the native Bitcoin asset.
+
+For definitions of the Bitcoin ledger and asset see link:./RFC-004-Bitcoin.md[RFC004].
+
+To fulfil the requirements of link:./RFC-003-SWAP-Basic.md[RFC003] this RFC defines:
+
+- The link:./RFC-003-SWAP-Basic.md#identity[identity] to be used when negotiating a SWAP on the Bitcoin ledger.
+- How to construct a Hash Time Lock Contract (HTLC) to lock the Bitcoin asset on the Bitcoin blockchain.
+- How to deploy, redeem and refund the HTLC during the execution phase of link:./RFC-003-SWAP-Basic.md[RFC003].
+
+== The Bitcoin Identity
+
+link:./RFC-003-SWAP-Basic.md[RFC003] requires ledgers have an *identity* type specified to negotiate a SWAP.
+
+The Identity to be used on Bitcoin is the 20-byte *pubkeyhash* which is the result of applying SHA-256 and then RIPEMD-160 to a user's compressed SECP256k1 public key.
+The compressed public key is used because it needs to be compatible with segwit transactions (see https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki#Restrictions_on_public_key_type)[BIP143]).
```

Comment:
```suggestion
The compressed public key is used because it needs to be compatible with segwit transactions (see link:https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki#Restrictions_on_public_key_type)[BIP143]).
```